### ISSUE TITLE
The Linux ZIP is broken.

### ISSUE THREAD

**Issue by pixelcmtd** [Amplus2/Amplissimus#43]
Just look at this `unzip` output:
```
Archive:  amplissimus-4.2.122-linux-x86_64.zip
   creating: tmp/amplissimus-4.2.122-linux-x86_64/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/cmake.check_cache  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/CMakeOutput.log  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/TargetDirectories.txt  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/rules.ninja  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/CMakeTmp/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/Amplissimus.dir/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/Amplissimus.dir/main.cc.o  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/Amplissimus.dir/my_application.cc.o  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/Amplissimus.dir/flutter/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/Amplissimus.dir/flutter/generated_plugin_registrant.cc.o  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/CompilerIdCXX/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/CompilerIdCXX/tmp/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/CompilerIdCXX/a.out  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/CompilerIdCXX/CMakeCXXCompilerId.cpp  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/CMakeSystem.cmake  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/CMakeCXXCompiler.cmake  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeFiles/3.20.1/CMakeDetermineCompilerABI_CXX.bin  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/.ninja_log  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/lib/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/lib/liburl_launcher_linux_plugin.so  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/lib/libflutter_linux_gtk.so  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/lib/libapp.so  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/icudtl.dat  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/version.json  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/NOTICES  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/fonts/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/fonts/MaterialIcons-Regular.otf  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/AssetManifest.json  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/assets/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/assets/logo.svg  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/assets/home.png  
 extracting: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/assets/settings.png
inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/data/flutter_assets/FontManifest.json  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/bundle/Amplissimus  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/install_manifest.txt  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/CMakeCache.txt  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/cmake_install.cmake  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/.ninja_deps  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/flutter/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/flutter/CMakeFiles/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/flutter/cmake_install.cmake  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/plugins/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/plugins/url_launcher_linux/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/plugins/url_launcher_linux/CMakeFiles/
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/plugins/url_launcher_linux/CMakeFiles/url_launcher_linux_plugin.dir/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/plugins/url_launcher_linux/CMakeFiles/url_launcher_linux_plugin.dir/url_launcher_plugin.cc.o  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/plugins/url_launcher_linux/liburl_launcher_linux_plugin.so  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/plugins/url_launcher_linux/cmake_install.cmake  
   creating: tmp/amplissimus-4.2.122-linux-x86_64/release/intermediates_do_not_run/
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/intermediates_do_not_run/Amplissimus  
  inflating: tmp/amplissimus-4.2.122-linux-x86_64/release/build.ninja
```

**Comment by pixelcmtd** [comment]
and we should also look though all of the other binaries too

**Comment by pixelcmtd** [comment]
it seems like only linux is bugged in this fashion

**Comment by pixelcmtd** [comment]
that one had another bug, fixed in d7c60d751592739f1804f18daffdb37a80c3dcf6
